src/products/repository/product-category.repository.ts

```diff
@@ -1,12 +1,32 @@
 import { Injectable } from "@nestjs/common";
 import { InjectRepository } from "@nestjs/typeorm";
 import { ProductCategory } from "../domain/product-category.entity";
-import { Repository } from "typeorm";
+import { DeleteResult, Repository } from "typeorm";
+import { UUID } from "crypto";
+import { validate } from "class-validator";
```

Comment:
**issue(non-blocking):** careful about unused imports

Reply:
Solved in a previous comment (above).